test/fixtures/example.css

```diff
@@ -34,7 +36,7 @@
         & .link {
             @mixin font-default;
             color: var(--color-red);
-            background: url('image.png');
+            background: var(--image-url);
```

Comment:
Can we use also the car in the image-set?

Reply:
yes, I'll change one of the `image-set` to reflect that